tee/nix/flake.nix

```diff
@@ -28,7 +28,7 @@
 
       zkOS-monorepo-source = builtins.fetchGit {
         url = "https://github.com/Cardinal-Cryptography/zkOS-monorepo.git";
-        rev = "da70131ba57ceb678a890c1b6e553710773686a8";
```

Comment:
Is it possible to extract this SHA to some variable that will be passed as command line argument in this script? And then call `nix  build -- abcdef1234567890 ` or, pass sha via env variable. Ideally, the SHA won't be required at all, ie once `git clone git@github.com:Cardinal-Cryptography/zkOS-monorepo.git` and nix will use already cloned sources. If this is not possible, then I'd advise to add another Makefile target in which it passes this SHA as env = `get rev-parse HEAD`.

Reply:
I don't know! I'll find out, but I'd propose moving that problem to another PR - no reason to hold your work until I figure that out.

Reply:
Created https://cardinal-cryptography.atlassian.net/browse/BAC-185